# TODO: which frame?
# Car state related
float64 	X 			# X position, [m]
float64 	Y 			# Y position, [m]
float64 	yaw 		# yaw angle, [rad]
float64     steering 	# steering angle, [rad]

# Car related
float64 	length_wheelbase 	# length of wheelbase, [m]
float64 	length_wheel_distance # length of wheel distance, [m]